guides/doc-Upgrading_and_Updating/topics/post_upgrade_template_checks.adoc

```diff
@@ -28,7 +28,7 @@ The following error occurs because of old syntax remaining in files after the up
 
 {Project} 6.4 onwards uses the `redhat_register` snippet instead of the `subscription_manager_registration` snippet.
 
-If you upgrade from {Project} 6.3 and earlier, ensure to replace the `subscription_manager_registration` snippet in your custom templates as follows:
+If you upgrade from {Project} 6.3 and earlier, ensure to replace the `subscription_manager_registration` snippet in your custom job or provisioning templates as follows:
```

Comment:
```suggestion
If you upgrade from {Project} 6.3 and earlier, you must replace the `subscription_manager_registration` snippet in your custom job or provisioning templates as follows:
```
Unrelated to your PR but I think this would be a better word choice.

Reply:
I agree. Thanks!